arraycontext/impl/pyopencl/__init__.py

```diff
@@ -161,15 +189,64 @@ def zeros(self, shape, dtype):
         import arraycontext.impl.pyopencl.taggable_cl_array as tga
         return tga.zeros(self.queue, shape, dtype, allocator=self.allocator)
 
-    def from_numpy(self, array: Union[np.ndarray, _ScalarLike]):
+    def empty_like(self, ary):
```

Comment:
Could you add a few container-using tests for these newly container-allowing functions?

Reply:
Added in 691c879.